server/container_create_linux.go

```diff
@@ -938,6 +944,25 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrfactory.Cont
 	return ociContainer, nil
 }
 
+func disableFipsForContainer(ctr ctrfactory.Container, containerDir string) error {
+	// Create a unique filename for the FIPS setting file.
+	fileName := filepath.Join(containerDir, "sysctl-fips")
+	content := []byte("0\n")
+
+	// Write the value '0' to disable FIPS directly to the file.
+	if err := os.WriteFile(fileName, content, 0o644); err != nil {
```

Comment:
BTW this should be 0o444 (as the original sysfs file is read-only for everyone).